### PR TITLE
Adding cephfs-top initiate testing

### PR DESCRIPTION
**Pull request by julpark-rh**
# Description

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-HS6BCE/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by neha-gangadhar** [comment]
@julpark-rh Please rebase PR.

File: tests/cephfs/cephfs_top/cephfs_top_initiate.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+log = Log(__name__)
+
+"""
+Testing description:
+
+Verify cephfs-top initial status and basic configuration
+
+Steps to Reproduce:
+1. Try to test install cephfs-top package
+2. Cehck if cephfs-top packages install is working well
```
Comment (sumabai) [inline tests/cephfs/cephfs_top/cephfs_top_initiate.py:17]: spell error

File: tests/cephfs/cephfs_top/cephfs_top_initiate.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+3. Check if cephfs-top is working without enabling mgr stats
+4. check if cephfs-top client is created well with appropriate permission
+5. check if cephfs-top -h,--help is working well
```
Comment (sumabai) [inline tests/cephfs/cephfs_top/cephfs_top_initiate.py:20]: There are some more steps in https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573829 that needs to be considered in the automation as below,

run "ceph auth ls" \| grep fstop | fstop client should be created after the installation
run "cephfs-top" in the client node | cephfs-top should be working
Create a client for non-default user | creating a client should be successful
run "cephfs-top --id <client>" | operation should be successful
Check if cephfs-top manual is present with detailed summary | all the summary should be listed with clear descriptions
Uninstall cephfs-top | Uninstall should be sucessful
run "cephfs-top" | Should not be working

Are these being covered in other automation?

File: tests/cephfs/cephfs_top/cephfs_top_initiate.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+Steps to Reproduce:
+1. Try to test install cephfs-top package
+2. Check if cephfs-top packages install is working well
+3. Check if cephfs-top is working without enabling mgr stats
+4. check if cephfs-top client is created well with appropriate permission
+5. check if cephfs-top -h,--help is working well
+"""
+
+
+def run(ceph_cluster, **kw):
```
Comment (sumabai) [inline tests/cephfs/cephfs_top/cephfs_top_initiate.py:24]: @julpark-rh This step seems missing in automation "Create a client for non-default user, run cephfs-top --id <client-name>" ,Can it be implemented?

**Review by neha-gangadhar — APPROVED** [review]
Approved

**Pull request merged**